src/libraries/System.Memory/tests/System.Memory.Tests.csproj

```diff
@@ -171,8 +171,6 @@
     <Compile Include="ReadOnlySpan\ToArray.cs" />
     <Compile Include="ReadOnlySpan\ToString.cs" />
     <Compile Include="ReadOnlySpan\ToUpperLower.cs" />
-    <Compile Include="$(CommonTestPath)Tests\System\StringTests.cs"
```

Comment:
We're just removing the file from the csproj and not moving it down to `System.Runtime.Tests` to avoid conflicts since I'll be putting up a PR soon to fix tests on Browser in that file.